.circleci/config.yml

```diff
@@ -3,28 +3,38 @@
 
 version: 2.1
 
-env: &env
-  TERM: dumb
-  CI: "true"
-  USER: circleci
-
-jobs:
-  build:
+executors:
+  chart-releaser:
     docker:
-    - image: cimg/base:2021.04
+       - image: quay.io/helmpack/chart-releaser:v1.5.0
     environment:
-      <<: *env
+      LC_ALL: "C.UTF-8"
+    resource_class: small
+
+commands:
+  build-cr:
     steps:
-      - checkout
       - run:
-         name: build
-         command: |
-           set -euo pipefail
+          command: apk add helm yq github-cli git jq openssh bash
+          name: Install dependencies for the chart publish image
+  deploy-cr:
+    steps:
+      - run:
+          command: ./scripts/chart_publish.sh $GH_TOKEN
+          name: Run the release script
 
-           echo "Success!"
+jobs:
+  release-chart:
+    executor: chart-releaser
+    steps:
+      - build-cr
+      - checkout
+      - deploy-cr
 
 workflows:
-  version: 2
-  build:
+  releaser:
     jobs:
-    - build
+      - release-chart:
```

Comment:
Shouldn't this also require an approval job? Or do we release on each main commit?

Reply:
Please refresh and check again @nicu-da  :)